fedora

```diff
@@ -24,7 +24,7 @@ fi
 RUN useradd -m -G wheel -u 1001 kokkos
 RUN echo '%wheel ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 USER kokkos
-ENV PATH=${INTEL:+/opt/intel/oneapi/compiler/latest/linux/bin:/opt/intel/oneapi/compiler/latest/linux/bin/intel64}${PATH:+:}${PATH}
-ENV LD_LIBRARY_PATH=${INTEL:+/opt/intel/oneapi/compiler/latest/linux/compiler/lib/intel64:/opt/intel/oneapi/mkl/latest/lib/intel64}${LD_LIBRARY_PATH:+:}${LD_LIBRARY_PATH}
+ENV PATH=${INTEL:+/opt/intel/oneapi/compiler/latest/linux/bin:/opt/intel/oneapi/compiler/latest/linux/bin/intel64}:${PATH:-}
+ENV LD_LIBRARY_PATH=${INTEL:+/opt/intel/oneapi/compiler/latest/linux/compiler/lib/intel64:/opt/intel/oneapi/mkl/latest/lib/intel64}:${LD_LIBRARY_PATH:-}
```

Comment:
Well, this syntax isn't quite right, as for empty `LD_LIBRARY_PATH` and empty `INTEL` the new value would be `:` instead of empty as it was previously.

Reply:
Sure, but that doesn't matter, does it? 
Anyway, I don't fell strongly abouth this pull request either way (since `docker` still complains).

Reply:
Not sure if `${var:-}` actually does anything.

https://docs.docker.com/reference/compose-file/interpolation/